README.md

```diff
@@ -3,6 +3,8 @@
 ## Developer environment
 
 1. To  test agaist a local instance of searchworks,  point your `settings.local.yml` to the local instance of SearchWorks.
+2. Install Gemfile `$ bundle install`
+3. Install required node modules (requires [node](https://nodejs.org/en/download) be installed) `npm install`.
```

Comment:
I think this should say `yarn install` (since we have a yarn.lock https://github.com/sul-dlss/sul-bento-app/blob/main/yarn.lock)

Reply:
Good catch. Thanks!